invenio_app_rdm/config.py

```diff
@@ -767,3 +783,17 @@ def files_rest_permission_factory(obj, action):
     ("invenio_app_rdm/default_static_page.html", "Default"),
 ]
 """List of available templates for pages."""
+
+FILES_REST_CHECKSUM_VERIFICATION_URI_PREFIXES = [
+    # 'root://eospublic'
+]
+"""URI prefixes of files their checksums should be verified"""
+
+FILES_INTEGRITY_REPORT_TEMPLATE = "invenio_app_rdm/emails/files_integrity_report.html"
```

Comment:
ping @anikachurilova in case you need to something with this new e-mail added (e.g. in CDS-RDM)
Is the path `.../email/...` coherent with any other path we have for e-mails templates? It would be nice to have the same folder name for all e-mails template.

Reply:
It was changed based on this comment https://github.com/inveniosoftware/invenio-app-rdm/pull/1975#discussion_r1034977373 from @slint

Reply:
discussed with @anikachurilova  IRL and we will follow the `flask-security` pattern `/templates/<module_name>/email/*.html`